udata_croquemort/templates/croquemort.html

```diff
@@ -0,0 +1,44 @@
+<script type="text/javascript">
+    const check_urls = {{ 'true' if config.CROQUEMORT else 'false' }};
```

Comment:
I'm not sure you can use that much ES6 without turning it to ES5 at some point.

Reply:
Yep, just a prototype ;-) Fetch is supported everywhere either.